src/components/PlayerCard/PlayerCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function PlayerCard({playerDataObj}) {
+    return (
+            <article>
+                <h3>{playerDataObj.name}</h3>
+                <h6>{playerDataObj.created_at}</h6>
+                <h5>Position: {playerDataObj.position}</h5>
```

Comment:
you can DRY this out a bit by destructuring your props like so: `PlayerCard({playerDataObj: { name, created_at, position}}) {` or...